src/DurableSDK/OrchestrationInvoker.cs

```diff
@@ -11,58 +11,97 @@ namespace Microsoft.Azure.Functions.PowerShellWorker.Durable
     using System.Linq;
     using System.Management.Automation;
 
-    using PowerShellWorker.Utility;
     using Microsoft.Azure.Functions.PowerShellWorker.Durable.Actions;
 
     internal class OrchestrationInvoker : IOrchestrationInvoker
     {
-        public Hashtable Invoke(OrchestrationBindingInfo orchestrationBindingInfo, IPowerShellServices pwsh)
+        private IExternalOrchestrationInvoker externalInvoker;
+        internal static string isOrchestrationFailureKey = "IsOrchestrationFailure";
```

Comment:
Wondering why not `const` (which would express the idea better, and would probably be slightly efficient than `static`)

Reply:
Yeah this should be `const`, I missed that during development.
Actually, I've moved this to our constant strings resource file, and changed all references to just read this value from there. Let me know if that seems reasonable to you!

Reply:
No big deal either way, but I would generally use string resources mostly for messages that may need to be localized, rephrased, etc. without changing functionality. String identifiers required for your code _correctness_ (is this the case here?) should stay in the code. This is arguable, so I will not insist.